src/core/agent/agent.types.ts

```diff
@@ -107,6 +107,15 @@ interface KeriNotification {
   a: Record<string, unknown>;
 }
 
+enum KeriConnectionType {
```

Comment:
@iFergal  We already have another `ConnectionType` enum. Should we keep it until everything related to DIDComm is deleted? I am afraid it will crash some things if we change that now.

Reply:
OK no problem!

@bao-sotatek if this merges first, in your PR can you rename this (and remove `Keri` prefixes from other interfaces if its not longer needed)